XcodeServerSDK/XcodeServerConfig.swift

```diff
@@ -8,13 +8,24 @@
 
 import Foundation
 
-public enum AvailabilityCheckState {
+public enum AvailabilityCheckState: Equatable {
     case Unchecked
     case Checking
     case Failed(NSError?)
     case Succeeded
 }
 
+/// Added `Equatable` to the enum to better test properties of this enum.
+public func == (a:AvailabilityCheckState, b:AvailabilityCheckState) -> Bool {
```

Comment:
in theory because they can hold values as properties swift can't determine how to compare unless explicitly told